loadtest/control/gencontroller/config.go

```diff
@@ -22,6 +22,8 @@ type Config struct {
 	NumChannelsPrivate int64 `default:"20" validate:"range:[0,]"`
 	// The target number of public channels to be created.
 	NumChannelsPublic int64 `default:"20" validate:"range:[0,]"`
+	// The target number of custom profile fields to be created.
+	NumCPAFields int64 `default:"2" validate:"range:[0,20]"`
```

Comment:
Is the upper limit a hard requirement?

Reply:
Yes, we currently only allow 20 custom profile properties.